apps/builder/app/builder/features/topbar/add-domain.tsx

```diff
@@ -94,56 +89,43 @@ export const AddDomain = ({
             </Label>
             <InputField
               id={id}
+              name="domain"
               autoFocus
               placeholder="your-domain.com"
-              value={domain}
-              disabled={
-                isPublishing || сreateState !== "idle" || domainState !== "idle"
-              }
+              disabled={isPending}
               onKeyDown={(event) => {
                 if (event.key === "Enter") {
-                  handleCreate();
+                  buttonRef.current
+                    ?.closest("form")
+                    ?.requestSubmit(buttonRef.current);
                 }
```

Comment:
Don't get it. Button is sibling so form should be an ancestor of event and enter should just work. Why manual handling?

Reply:
See formAction we are not submitting default form action

Reply:
gotcha